samples/nrf_profiler/sample.yaml

```diff
@@ -9,5 +9,10 @@ tests:
       - nrf52840dk_nrf52840
       - nrf9160dk_nrf9160_ns
       - nrf21540dk_nrf52840
+    platform_allow:
+      - nrf52dk_nrf52832
+      - nrf52840dk_nrf52840
+      - nrf9160dk_nrf9160_ns
+      - nrf21540dk_nrf52840
```

Comment:
Everywhere else `platform_allow` is above `integration_platforms`. Here, order is reversed.

Reply:
good catch:) my mistake. there is no required order anyhow nor I see a point in recommending any. I see it is not a blocker for you. Therefore, I won't change it since it affects nothing and would just cause CI to restart and dismiss reviews.